dev-util/gengetopt/gengetopt-2.23.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="A tool for generating scanners"
+DESCRIPTION="Gengetopt is a tool to write command \
+line option parsing code for C programs."
+HOMEPAGE="https://www.gnu.org/software/gengetopt/"
+COPYRIGHT="1999-2007 Free Software Foundation, Inc."
```

Comment:
For copyright check the authors file, the years for the copyright could be tracked at git